src/index.html

```diff
@@ -11,6 +11,36 @@
     <link rel="stylesheet" href="./style.css">
   </head>
   <body>
-    <h1>Search bar airbnb</h1>
+    <div class="container">
+
+      <form
+        action="#"
+        method="post"
+        name="form"
+        data-qa="big"
+        class="searchbar">
```

Comment:
Need to fix it everywhere
```suggestion
      <form
        action="#"
        method="post"
        name="form"
        data-qa="big"
        class="searchbar"
      >
```